server/manager/src/main/java/org/apache/accumulo/manager/tableOps/merge/ReserveTablets.java

```diff
@@ -71,7 +71,7 @@ public long isReady(FateId fateId, Manager env) throws Exception {
     int locations = 0;
     int wals = 0;
 
-    var startTime = NanoTime.now();
+    var startTime = Timer.startNew();
```

Comment:
Later in the class `startTime.elapsed().toMillis()` is called, could use the new TimeUnit version of elapsed.

Reply:
In 453fa35 I went through and replaced the few places where the returned value of `elapsed()` was immediately being converted.